### ISSUE TITLE
Slowing down the animated GIF demo

### ISSUE THREAD

**Issue by mirekphd** [heartexlabs/label-studio#4803]
**Is your feature request related to a problem? Please describe.**
It's hard to follow what happens in the GIF (and it oversells the actual annotation speed as well:)

**Describe the solution you'd like**
Slower down animation (possibly split into several ones if there are some limits).

**Describe alternatives you've considered**
None.

**Additional context**
None

**Comment by caitlinwheeless** [comment]
@mirekphd could you link to the page that has the gif?

**Comment by EuphoriaCelestial** [comment]
> @mirekphd could you link to the page that has the gif?

https://raw.githubusercontent.com/heartexlabs/label-studio/master/images/annotation_examples.gif
I think he means this GIF.
I find it a bit hard to follow up too, but its not really a problem
